libnmstate/__init__.py

```diff
@@ -36,3 +41,12 @@
     "schema",
     "PrettyState",
 ]
+
+
+def get_version():
```

Comment:
This function should be prefixed with `_` since it is not meant to be part of the API.

Reply:
Done!